com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -93,6 +108,8 @@ Jaedyn:
 
 Samuel:
 
+- [92fc3e6d] (2021-09-20) Sam Bellomo / feat: transform teleport (#1200)
+- [d8a3a9b3] (2021-09-20) Sam Bellomo / feat: ClientNetworkTransform (#1177)
 - [5513c906] (2021-09-16) Sam Bellomo / feat: interpolation for network transform (#1060)
 - [cc7a7d5c] (2021-08-19) Sam Bellomo / test: adding more details to multiprocess readme (#1050)
 - [13e2b7f1] (2021-07-13) Sam Bellomo / test: multiprocess tests part 6: fixing issues runnings all tests together (#957)
```

Comment:
if changes are not user facing, do we still need to document them here?

Reply:
no. the goal is to document user facing and effecting changes. any behavioral changes users need to know about or any public API changes etc should be documented here. internal, not user facing/effecting changes, chores etc does not have to be documented in the changelog. you might take a look at other entries in the list :)

Reply:
ok, then we'd need to review this list, all the multiprocess tests stuff isn't userfacing.
There's also a few items that are more internal iteration that users won't care about (they'll only care about the end product, not how we got there)